design-docs/CLI-Orders.md

```diff
@@ -116,47 +185,34 @@ Usage: planet orders wait [OPTIONS] [ORDER_ID]
   This command outputs the order details at the end of the wait, optionally
   pretty-printed.
 
-Options:
-  --state TEXT   Stop waiting when order reaches this state
-  --period TEXT  Duration of time for one polling cycle, in seconds.
-  --pretty       Format JSON output.
-  --help         Show this message and exit.
-```
+Options:  
+  --state TEXT   Stop waiting when order reaches this state  
+  --period TEXT  Duration of time for one polling cycle, in seconds.  
+  --pretty       Format JSON output.  
+
 
 ### Usage Examples
 
-User Story: As a CLI user I would like to wait for an order to be ready for 
+User Story: As a CLI user I would like to wait for an order to be ready for
 download so I can send a message to another machine to do the download.
 
 `$ planet orders wait 65df4eb0-e416-4243-a4d2-38afcf382c30`
 
-User Story: As a CLI user I would like to wait for an order to be ready for 
+User Story: As a CLI user I would like to wait for an order to be ready for
 download and then I would like to download the order.
 
 ```
 $ planet orders wait 65df4eb0-e416-4243-a4d2-38afcf382c30 \
-&& planet orders download 65df4eb0-e416-4243-a4d2-38afcf382c30 
+&& planet orders download 65df4eb0-e416-4243-a4d2-38afcf382c30
 ```
 
-User Story: As a CLI user I would like to create an order, wait for it to be 
```

Comment:
I think this example is wrong as `planet orders wait` and `planet orders download` do not support stdin. So, I just removed it.